modules/dnn/CMakeLists.txt

```diff
@@ -295,6 +295,13 @@ if(TARGET ocv.3rdparty.cann AND OPENCV_TEST_DNN_CANN)
   endif()
 endif()
 
+ocv_option(OPENCV_TEST_DNN_TIMVX "Build test with TIM-VX" (HAVE_TIMVX))
+if(OPENCV_TEST_DNN_TIMVX)
+  if(TARGET opencv_test_dnn)
+  ocv_target_compile_definitions(opencv_test_dnn PRIVATE "HAVE_TIMVX=1")
```

Comment:
indentation

Reply:
Thank you for finding this! Meanwhile please hold merging this PR until the hardware is in the CI running the tests.

Reply:
Resolved.